src/utils/hd-wallet.ts

```diff
@@ -23,27 +27,27 @@ export function derivePathFromKey (path, key) {
   return segments.reduce((parentKey, segment) => deriveChild(parentKey, segment), key)
 }
 
-export function derivePathFromSeed (path, seed) {
+export function derivePathFromSeed (path: string, seed: string|Buffer) {
   if (!['m', 'm/'].includes(path.slice(0, 2))) {
     throw new InvalidDerivationPathError()
   }
   const masterKey = getMasterKeyFromSeed(seed)
   return derivePathFromKey(path.slice(2), masterKey)
 }
 
-function formatAccount (keys) {
+function formatAccount (keys: {secretKey: Uint8Array, publicKey: Uint8Array}) {
```

Comment:
I think we should add return types for functions (on the other ones too). For example here, publicKey is marked as `any` (the returned object)

```suggestion
function formatAccount (keys: {secretKey: Uint8Array, publicKey: Uint8Array}):{
  secretKey: string;
  publicKey: string;
} {
```
Function return typing motivation: https://stackoverflow.com/a/53666026/10535444 (tl;dr strict)